apps/bfd-model/pom.xml

```diff
@@ -37,11 +37,20 @@
 
 			<dependency>
 				<!-- Used to run our unit and integration tests. -->
-				<groupId>junit</groupId>
-				<artifactId>junit</artifactId>
-				<version>4.13.1</version>
+				<groupId>org.junit.jupiter</groupId>
+				<artifactId>junit-jupiter-engine</artifactId>
+				<version>${junit.version}</version>
 				<scope>test</scope>
 			</dependency>
+
+			<dependency>
+				<!-- Allows junit compatibility with old junit4 tests -->
+				<groupId>org.junit.vintage</groupId>
+				<artifactId>junit-vintage-engine</artifactId>
+				<version>${junit.vintage.version}</version>
+				<scope>test</scope>
+			</dependency>
```

Comment:
we shouldn't need any junit stuff here since we already have it in dependencyManagement in the root pom